content/docs/traffic-management/health-checks/backend.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Backend health checks
+weight: 20
+---
+
+Automatically monitor the status of backends by configuring health checks.
```

Comment:
Can you set a healthcheck for non-Backend components like the control plane? idk if that is outside the scope of this

Reply:
It looks like the dev PR uses an HTTPListenerPolicy to set a healthcheck on the gateway, https://github.com/kgateway-dev/kgateway/pull/11629

maybe that would be a separate topic or H2?

Reply:
That's the gateway.md page